Java/GCD_algo.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class GCD {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		Scanner sc = new Scanner(System.in);
+		int a = sc.nextInt();
+		int b = sc.nextInt();
+		int gcd = 0;
+		
+		
+		if (a>b)
+		{
```

Comment:
Consider reading this before you commit to VCS: https://google.github.io/styleguide/javaguide.html